### PR TITLE
Updated Italian translation for welcome gramplet

### PR DESCRIPTION
**Pull request by giansalvo**
Updated Italian translation of Gramps Welcome gramplet

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -7934,7 +7935,7 @@ msgstr ""
 
 #: ../gramps/gen/lib/ldsord.py:105
 msgid "Canceled"
-msgstr ""
+msgstr "Annullato"
```
Comment (tosky) [inline po/it.po]: Do you have a specific reference that says that the LDS church uses this translation? If not, please leave this translation out from this change.

File: po/it.po [hunk 2]
```diff
@@ -7965,7 +7966,7 @@ msgstr "Non ridimensionare l'albero"
 
 #: ../gramps/gen/lib/ldsord.py:110
 msgid "Infant"
-msgstr ""
+msgstr "Bambino"
```
Comment (tosky) [inline po/it.po]: See above

File: po/it.po [hunk 3]
```diff
@@ -23389,6 +23390,10 @@ msgid ""
 "powerful features.\n"
 "\n"
 msgstr ""
+"Gramps è un'applicazione progettata per ricerche genealogiche. Anche se "
+"simile ad altri programmi di genealogia, Gramps offre alcune utili e potenti "
```
Comment (tosky) [inline po/it.po]: it's "unique and poweful", not "useful and powerful"

File: po/it.po [hunk 4]
```diff
@@ -23427,6 +23432,13 @@ msgid ""
 "powerful, yet easy to use.\n"
 "\n"
 msgstr ""
+"Gramps è stato creato da genealogisti per genealogisti, organizzati nel "
+"Progetto Gramps. Gramps è un pacchetto software Libero (Open Source). Ciò "
+"vuol dire che sei libero di farne copie e distribuirlo a chiunque tu voglia. "
+"Gramps è sviluppato e  curato da un gruppo di volontari sparsi per il mondo, "
```
Comment (tosky) [inline po/it.po]: extra space ("e  curato")

File: po/it.po [hunk 5]
```diff
@@ -23440,6 +23452,13 @@ msgid ""
 "For more details, please read the information at the links above\n"
 "\n"
 msgstr ""
+"La prima cosa che devi fare è creare un nuovo Albero Genealogico. Per creare "
```
Comment (tosky) [inline po/it.po]: albergo genealogico should be lowercase, the Camel Case Writing Is English Only (and not always).

File: po/it.po [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # Italian translation for GRAMPS
 # This file is distributed under the same license as the Gramps package.
-# Copyright (C) 2001-2013 the Gramps Project
+# Copyright (C) 2001-2019 the Gramps Project
```
Comment (tosky) [inline po/it.po]: Please submit the change against the maintenance/gramps50 branch, not master.

But first I'm going to restore the existing translation of the gramplet, which already existed as a single message and you can still see in the comments in the po file, but it was split out in several messages.
After that, this review may not be needed.

File: po/it.po [hunk 7]
```diff
@@ -23456,6 +23475,16 @@ msgid ""
 "can also drag the Properties button to reposition the gramplet on this page, "
 "and detach the gramplet to float above Gramps."
 msgstr ""
+"In questo momento stai leggendo dalla vista \"Cruscotto\", dove puoi "
+"aggiungere le gramplet di tua scelta. Puoi anche aggiungere le gramplet a "
+"qualsiasi tipo di vista aggiungendo una barra laterale o una barra in basso "
+"e cliccando col tasto destro a destra della scheda.\n"
+"\n"
+"Cliccando sull'icona di configurazione nella barra degli strumenti puoi "
```
Comment (tosky) [inline po/it.po]: Facendo clic

File: po/it.po [hunk 8]
```diff
@@ -23456,6 +23475,16 @@ msgid ""
...
+"anche aggiungere colonne addizionali, mentre cliccando col tasto destro "
```
Comment (tosky) [inline po/it.po]: see above

**Comment by tosky** [comment]
All the strings have been restored (on the 5.0 branch, which is periodically merged into master); this change is no more required, but thanks for pointing out that few important strings temporarily disappeared.

File: po/it.po (hunk 6 above)
Comment (giansalvo) [reply po/it.po]: Hi tosky,
thanks for your review and for pointing out the maintenance/gramps50 branch.

**Pull request closed without merging**